src/frontend/components/UI/AccountDropdown/index.tsx

```diff
@@ -1,40 +1,42 @@
 import { Menu } from '@mantine/core'
-import React, { useContext } from 'react'
+import React from 'react'
+import walletStore from 'frontend/state/WalletState'
```

Comment:
can we call it walletState? we have too many things called store

Reply:
sure, just copied paste from another component 😅

Reply:
done